packages/api/src/dao/EpochDAO.js

```diff
@@ -18,6 +18,12 @@ module.exports = class EpochDAO {
       timestamp: currentBlock.header.timestamp
     })
 
+    const bestValidator = this.knex('blocks')
```

Comment:
This does not look like a best validator belonging to the epoch

Reply:
✅